app_test.py

```diff
@@ -33,11 +33,32 @@ def test_index(self):
         response = self.client.get('/', 'http://pytest.local')
         self.assertEqual(response.status_code, 200)
 
+    def test_info_index(self):
+        """ Check the info index path responds with a 301 """
+        response = self.client.get('/info')
+        self.assertEqual(response.status_code, 301)
+
     def test_info_help(self):
         """ Check the info help path responds with 200 OK """
         response = self.client.get('/info/help')
         self.assertEqual(response.status_code, 200)
 
+    def test_info_resources(self):
+        """ Check the info resources path responds with 200 OK """
```

Comment:
These docstrings should have no spaces surrounding them, and end with a period.

Reply:
The entire test file is like that, isn't it?